package.json

```diff
@@ -117,6 +117,7 @@
     "react-router": "^2.2.0",
     "react-router-redux": "^4.0.0",
     "redux": "^3.0.0",
+    "redux-form": "^5.3.1",
```

Comment:
Let's stick with the stable version for now. There's no great benefit of moving to an alpha for a kickstarter project. We can upgrade when 6 is the stable release